src/stirling/source_connectors/socket_tracer/uprobe_manager.cc

```diff
@@ -730,9 +892,14 @@ void UProbeManager::DeployUProbes(const absl::flat_hash_set<md::UPID>& pids) {
   int uprobe_count = 0;
 
   uprobe_count += DeployOpenSSLUProbes(proc_tracker_.new_upids());
+  uprobe_count += DeployGrpcCUProbes(proc_tracker_.new_upids());
+
   if (FLAGS_stirling_rescan_for_dlopen) {
-    uprobe_count += DeployOpenSSLUProbes(PIDsToRescanForUProbes());
+    auto pids_to_rescan_for_uprobes = PIDsToRescanForUProbes();
+    uprobe_count += DeployOpenSSLUProbes(pids_to_rescan_for_uprobes);
+    uprobe_count += DeployGrpcCUProbes(pids_to_rescan_for_uprobes);
```

Comment:
@orishuss Seems this one is not addressed yet?

Reply:
@orishuss Let's also make the flag value default for false for now.

After some thoughts, I think it makes more sense to change the flag value to true, after specific performance regression tests with specific gRPC python app. Since the existing performance regression tests do not have coverage on python gRPC.

This also makes this PR less risky, and this PR is ready to land after fixing the gcc build and changing the flag default value.

Reply:
Done